kevm-pyk/src/tests/unit/test_foundry_list.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING
+
+import pytest
+from pyk.proof import APRProof
+
+from tests.utils import REPO_ROOT
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Final
+
+TEST_DATA = REPO_ROOT / 'kevm-pyk/src/tests/unit/test-data'
```

Comment:
Create a file `utils.py` in `src/test/unit`. There, define `TEST_DATA_DIR` relative to that file. (See `integration/utils.py` for an example.)

Reply:
Much better, thank you !